Testing/SystemTests/tests/framework/Peak2ConvCell_Test.py

```diff
@@ -1,971 +0,0 @@
-# Mantid Repository : https://github.com/mantidproject/mantid
-#
-# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
-#   NScD Oak Ridge National Laboratory, European Spallation Source,
-#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
-# SPDX - License - Identifier: GPL - 3.0 +
-# pylint: disable=invalid-name,no-init,too-many-arguments,too-many-branches, unused-variable,too-many-return-statements
-# This script creates numerous PeaksWorkspaces for different Crystal Types and Centerings. Random errors
-# are also introduced into the peak's.  Each PeaksWorkspace is sent through the algorithm's FindPeaksMD,
-# FindUBUsingFFT, and SelectByForm to determine the corresponding Primitive and Conventional cells. These
-# results are tested against the theoretical results that should have been gotten
-
-# NOTE; THIS TEST TAKES AN EXTREMELY LONG TIME. DELETE "XXX" IN requiredFiles method to get it to run.
-# !!!!!!!!!  REPLACE THE "XXX" OR else !!!!!!!!!!
-
-
-# import systemtesting
-import numpy
-from numpy import matrix
-import math
-import random
-import mantid
-from mantid.api import mtd
-from mantid.simpleapi import (
-    CopySample,
-    CreatePeaksWorkspace,
-    CreateSingleValuedWorkspace,
-    FindUBUsingFFT,
-    IndexPeaks,
-    LoadInstrument,
-    SelectCellWithForm,
-)
-
-
-# from mantid.simpleapi import *
-# TODO premultiply cases, fix up.. Maybe not needed Cause Conv cell was "Nigglied"
-# TODO: SWitch cases, if use approx inequality, may get error cause low level code
-# [does Not](does) premult but when it [should](should not)
-class Peak2ConvCell_Test(object):  # (systemtesting.MantidSystemTest):
```

Comment:
Why did you decide to remove this rather than inherit from the sytem test class?

Reply:
The main reason was because there hasn't been a meaningful change in the file for 7 years and its unclear if the test has ever been run as a system test in our CI. There is also a nested for loop at the end of the file with 8 loops in `runTest`, which I was scared to bring to life